synapse/api/auth.py

```diff
@@ -482,8 +484,14 @@ async def get_user_by_access_token(
             TypeError,
             ValueError,
         ) as e:
-            logger.warning("Invalid macaroon in auth: %s %s", type(e), e)
-            raise InvalidClientTokenError("Invalid macaroon passed.")
+            logger.warning(
+                "Invalid access token in auth: %s %s. (Neither a known token nor a valid macaroon.)",
```

Comment:
Would it be simpler to just say it is not a known access token and not mention macaroons at all?

```suggestion
                "Invalid access token in auth: %s %s.",
```

I'm OK either way, but figured I'd suggest it!

Reply:
Seems reasonable.